src/components/Navbar.js

```diff
@@ -1,123 +1,135 @@
-import React from "react";
-import { Link } from "gatsby";
+import React, { useState } from "react";
+import { Link, useStaticQuery, graphql } from "gatsby";
+import Img from "gatsby-image";
 import headerBackground from "../img/header-bg.png";
-import logoImg from "../img/header-rosie.png";
-import logoImgMobile from "../img/header-mobile.png";
-import maskNowImg from "../img/header-masksnow.png";
 
 const HamburgerLine = () => {
   return <span />;
 };
 
-const Navbar = class extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      active: false,
-      navBarActiveClass: ""
-    };
-  }
+const Navbar = () => {
+  const data = useStaticQuery(graphql`
+    query {
+      rosie: file(relativePath: { eq: "header-rosie.png" }) {
+        childImageSharp {
+          fixed(height: 300) {
+            ...GatsbyImageSharpFixed
+          }
+        }
+      }
+      mobile: file(relativePath: { eq: "header-mobile.png" }) {
+        childImageSharp {
+          fixed(height: 300) {
+            ...GatsbyImageSharpFixed
+          }
+        }
+      }
+      maskNow: file(relativePath: { eq: "header-masksnow.png" }) {
+        childImageSharp {
+          fluid {
+            ...GatsbyImageSharpFluid
+          }
+        }
```

Comment:
in the ui as it stands - the masksNow logo scales smoothly and then swaps with the mobile version. This one kind of - pops from one size to another. I tried using maxWidth param for `fluid` but it didn't help. Feel free to try to tweak this to work as it is on the site currently or in the interest of time - we can leave _this_ image unoptimized.  The rosie image optimization looks great!